migrations/20201125113750-migrate-collective-policies-to-html.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
+
+import showdown from 'showdown';
+import { buildSanitizerOptions, sanitizeHTML } from '../server/lib/sanitize-html';
+
+const converter = new showdown.Converter();
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    const collectivesWithExpensePolicy = await queryInterface.sequelize.query(
+      `
+        SELECT "id", "expensePolicy" FROM "Collectives"
+        WHERE LENGTH("expensePolicy") > 0 AND "expensePolicy" NOT LIKE '<%'
+        AND "deletedAt" IS NULL;
+      `,
+      { type: Sequelize.QueryTypes.SELECT },
+    );
+
+    const collectivesWithContribPolicy = await queryInterface.sequelize.query(
+      `
+        SELECT "id", "contributionPolicy" FROM "Collectives"
+        WHERE LENGTH("contributionPolicy") > 0 AND "contributionPolicy" NOT LIKE '<%'
+        AND "deletedAt" IS NULL;
+      `,
+      { type: Sequelize.QueryTypes.SELECT },
+    );
+
+    const sanitizeOptions = buildSanitizerOptions({
+      basicTextFormatting: true,
```

Comment:
I forgot to mention it on the issue, but we also need `multilineTextFormatting` since the editor allows lists

Reply:
updated 👍🏻